runtime/src/assets_config.rs

```diff
@@ -0,0 +1,116 @@
+    pub const AssetAccountDeposit: Balance = deposit(1, 16);
+    pub const ApprovalDeposit: Balance = EXISTENTIAL_DEPOSIT;
+    pub const AssetsStringLimit: u32 = 50;
+    /// Key = 32 bytes, Value = 36 bytes (32+1+1+1+1)
+    // https://github.com/paritytech/substrate/blob/069917b/frame/assets/src/lib.rs#L257L271
+    pub const MetadataDepositBase: Balance = deposit(1, 68);
+    pub const MetadataDepositPerByte: Balance = deposit(0, 1);
+}
+
+parameter_types! {
+    pub NftsPalletFeatures: PalletFeatures = PalletFeatures::all_enabled();
+    pub const NftsCollectionDeposit: Balance = UNIT / 10;
+    pub const NftsItemDeposit: Balance = UNIT / 1_000;
+    pub const NftsMetadataDepositBase: Balance = deposit(1, 129);
+    pub const NftsAttributeDepositBase: Balance = deposit(1, 0);
+    pub const NftsDepositPerByte: Balance = deposit(0, 1);
+    pub const NftsMaxDeadlineDuration: BlockNumber = 12 * 30 * DAYS;
+}
+
+impl pallet_nfts::Config for Runtime {
+    type RuntimeEvent = RuntimeEvent;
+    type CollectionId = CollectionId;
+    type ItemId = ItemId;
+    type Currency = Balances;
+    type CreateOrigin = AsEnsureOriginWithArg<EnsureSigned<AccountId>>;
+    type ForceOrigin = AssetsForceOrigin;
+    type Locker = ();
+    type CollectionDeposit = NftsCollectionDeposit;
+    type ItemDeposit = NftsItemDeposit;
+    type MetadataDepositBase = NftsMetadataDepositBase;
+    type AttributeDepositBase = NftsAttributeDepositBase;
+    type DepositPerByte = NftsDepositPerByte;
+    type StringLimit = ConstU32<256>;
+    type KeyLimit = ConstU32<64>;
+    type ValueLimit = ConstU32<256>;
+    type ApprovalsLimit = ConstU32<20>;
+    type ItemAttributesApprovalsLimit = ConstU32<30>;
+    type MaxTips = ConstU32<10>;
+    type MaxDeadlineDuration = NftsMaxDeadlineDuration;
+    type MaxAttributesPerCall = ConstU32<10>;
+    type Features = NftsPalletFeatures;
+    type OffchainSignature = Signature;
+    type OffchainPublic = <Signature as Verify>::Signer;
+    type WeightInfo = pallet_nfts::weights::SubstrateWeight<Self>;
+    #[cfg(feature = "runtime-benchmarks")]
+    type Helper = ();
+}
+
+parameter_types! {
+    pub const NftFractionalizationPalletId: PalletId = PalletId(*b"fraction");
+    pub NewAssetSymbol: BoundedVec<u8, AssetsStringLimit> = (*b"FRAC").to_vec().try_into().unwrap();
+    pub NewAssetName: BoundedVec<u8, AssetsStringLimit> = (*b"Frac").to_vec().try_into().unwrap();
+}
+
+impl pallet_nft_fractionalization::Config for Runtime {
+    type RuntimeEvent = RuntimeEvent;
+    type Deposit = AssetDeposit;
+    type Currency = Balances;
+    type NewAssetSymbol = NewAssetSymbol;
+    type NewAssetName = NewAssetName;
+    type StringLimit = AssetsStringLimit;
+    type NftCollectionId = <Self as pallet_nfts::Config>::CollectionId;
+    type NftId = <Self as pallet_nfts::Config>::ItemId;
+    type AssetBalance = <Self as pallet_balances::Config>::Balance;
```

Comment:
Net result is the same, but this should be amended to `<Self as pallet_assets::Config<TrustBackedAssets>>::Balance`. This has long bugged me. 🤣